etc-conf/rhsm.conf

```diff
@@ -94,6 +94,12 @@ autoAttachInterval = 1440
 splay = 1
 # If set to 1, rhsmcertd will not execute.
 disable = 0
+# Set to 1, when rhsmcertd will try to do automatic registration.
```

Comment:
I'd suggest we change this line to "Set to 1 when rhsmcertd should attempt automatic registration"